### PR TITLE
Integrate HTML report with Composer!

### LINKED ISSUE #11: WIP: HTML Report
Sketch

sketch

Next step: collect links to good examples of coloring, fonts and overall web page styles.
Next step: discuss that with our Web Frontend Team.

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Closes #11!

### CODE REVIEW

File: composer/src/main/kotlin/com/gojuno/composer/html/HtmlReport.kt [hunk 1]
```diff
@@ -20,18 +24,62 @@ fun writeHtmlReport(gson: Gson, suites: List<Suite>, outputDir: File): Completab
...
                 .tests
                 .map { it to File(File(suitesDir, "$suiteId"), it.adbDevice.id).apply { mkdirs() } }
-                .map { (test, testDir) -> test.toHtmlFullTest(htmlReportDir = testDir) to testDir }
-                .forEach { (htmlFullTest, testDir) -> File(testDir, "${htmlFullTest.id}.json").writeText(gson.toJson(htmlFullTest)) }
+                .map { (test, testDir) -> Triple(test, test.toHtmlFullTest(suiteId = "$suiteId", htmlReportDir = testDir), testDir) }
+                .forEach { (test, htmlTest, testDir) ->
+                    val testJson = gson.toJson(htmlTest)
+                    val testHtmlFile = File(testDir, "${htmlTest.id}.html")
+
+                    testHtmlFile.writeText(indexHtml
```
Comment (yunikkk) [inline composer/src/main/kotlin/com/gojuno/composer/html/HtmlReport.kt]: Maybe extract block into private helper that writes htmlTest to file?

File: composer/src/main/kotlin/com/gojuno/composer/html/HtmlReport.kt [hunk 2]
```diff
@@ -40,3 +88,31 @@ fun writeHtmlReport(gson: Gson, suites: List<Suite>, outputDir: File): Completab
...
+            .map { line -> """<div class="log__${cssClassForLogcatLine(line)}">${StringEscapeUtils.escapeXml11(line)}</div>""" }
+            .fold(StringBuilder("""<div class="content"><div class="card log">""")) { stringBuilder, line ->
+                stringBuilder.appendln(line)
+            }
+            .let { it.appendln("""</div></div>""") }
+            .let { it.toString() }
+}
+
+fun cssClassForLogcatLine(logcatLine: String): String {
+    // Logcat line example: `06-07 16:55:14.490  2100  2100 I MicroDetectionWorker: #onError(false)`
```
Comment (yunikkk) [inline composer/src/main/kotlin/com/gojuno/composer/html/HtmlReport.kt]: The logcat line in comment actually contain date and id-s in the beginning, not loglevel.

File: composer/src/main/kotlin/com/gojuno/composer/html/HtmlReport.kt (hunk 1 above)
Comment (artem-zinnatullin) [inline composer/src/main/kotlin/com/gojuno/composer/html/HtmlReport.kt]: Let's do it in a separate PR, I also want to make it parallel via Rx
